### DISCUSSION TITLE
dp.Text Markdown with specific characters (french) doesn't work

### DISCUSSION THREAD

**Discussion by Bebio95** [datapane/datapane#115] [Help] [answered]
Hello, 

I have a markdown file (Test.md) with some french specific characters inside : _modèle opérationnel_

My code : 

```
import datapane as dp
Report = dp.Report(dp.Text(file="C:/Users/a61787/Documents/Test.md"))
Report.save(path='Test.html', open=True)
```

which gives me : 

modÃ¨le opÃ©rationnel

Is there a way to handle this ? specifying an encoding type maybe ?
Thanks for any help

**Comment by mands** [comment, marked as answer]
Hi @Bebio95 - I imagine this may be an encoding issue. What OS are you running on, are you on Windows? We've had some other language encoding issues so will look at fixing asap.

Thanks!

**Reply by Bebio95** [reply]
Hello, 
Indeed, I'm on Windows 7. Thanks for your answer.
Looking forward for next improvements !

**Reply by mands** [reply]
Thanks for the feedback - that helps a lot! Hope to have a fix/workaround out shortly!

**Reply by mands** [reply]
Hi @Bebio95 ,

We've just pushed a fix for this that will be released on pip as 0.12.0 in a few days. Please see #142 for a detailed discussion, but basically seems to be an encoding mismatch between the file and the default Python read behaviour.

**Reply by Bebio95** [reply]
Hi, 
Great news ! 
Thanks a lot for letting me know.
